api/gettime.php

```diff
@@ -45,31 +49,27 @@
     if ($isLoggedIn) {
       if($to_date!="''")
       {
-        $record = ttTimeHelper::getAllDateRecords($from_date,$to_date );
-        foreach ($record as $rec)
-        {
-          $success_response = [$rec];
-          $response = json_encode($success_response);
-          print_r($response);
-        }
+        $record = ttTimeClassHelper::getAllDateRecords($from_date,$to_date );
+        $success_response = ['success'=>'true','data'=>$record];
+        $response = json_encode($success_response);
+        print_r($response);
+
       }
       //if no values entered
       else
       {
         $to_date=$from_date;
-        $record = ttTimeHelper::getAllDateRecords($from_date,$to_date);
+        $record = ttTimeClassHelper::getAllDateRecords($from_date,$to_date);
```

Comment:
Why call `ttTimeClassHelper::getAllDateRecords($from_date,$to_date);` twice?

Reply:
different case values are passed in the parameters, which depends on the if-else condition.
if the function is called only once globally. it might return wrong values when called inside the if-else condition.

Reply:
I know.  When 

if($to_date=="''")
{
$to_date=$from_date;
}
and you dont need the else part

Reply:
edited now sir,
sorry about that.